scalableminds/webknossos
# Restore Team Assignments for Datasets

### Mailable description of changes:
- [Dashboard.Advanced Datasets View] Restored Team Assignments for Datasets
- Click on one of the existing teams to toggle a team assignment modal

### Steps to test:
- Go to Dashboard.Advanced Datasets View
- Click on one of the existing teams to toggle a team assignment modal
- Add a new team and click ok --> Table should update
- Try removing a team
- Try removing the `owner` team (aka Connectomics). Should be impossible.

### Issues:
- fixes #2020

------
